CefSharp/CEFCursorType.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright © 2010-2015 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+namespace CefSharp
+{
+    public enum CefCursorType
+    {
+        CT_POINTER = 0,
```

Comment:
Can you remove the `CT_` prefix and make these proper case. Bring them inline with `.Net` styling.
e.g. `CT_POINTER` becomes `Pointer`